src/contacts/model/ContactModel.ts

```diff
@@ -183,7 +185,8 @@ export class ContactModel {
 			}
 		}
 
-		if (isApp()) {
+		const isAllowedToSync = deviceConfig.getUserSyncContactsWithPhonePreference(locator.logins.getUserController().userId)
```

Comment:
bad idea to import `deviceConfig` and `locator` directly into the model. They should be injected and this is yet another argument to move the sync stuff elsewhere.

Reply:
this is still not addressed